common-ui/src/main/res/values/styles.xml

```diff
@@ -136,6 +136,16 @@
         <item name="android:layout_height">1dp</item>
     </style>
 
+    <style name="OverflowMenuDivider">
```

Comment:
I was not planning to add this style in this PR, but since you saw it in our pairing session, I think is ok if you take a look and we merge it. (I think you wanted to do something with it)

Reply:
ditto